<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="HW2247" name="com.web.poc.script.GS1_Login" tests="2" failures="0" timestamp="10 Oct 2017 11:16:24 GMT" time="2.809" errors="1">
  <testcase name="beforeTest" time="2.809" classname="com.web.poc.script.GS1_Login">
    <error type="java.io.FileNotFoundException" message="C:\Users\e001207\Documents\GS1\GS1\GS1\testdata\JiraMapping.csv (The system cannot find the file specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\e001207\Documents\GS1\GS1\GS1\testdata\JiraMapping.csv (The system cannot find the file specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileReader.<init>(Unknown Source)
	at com.web.automation.testmanagement.JiraIntegration.LoadJiraMappingSheet(JiraIntegration.java:173)
	at com.web.automation.accelerators.TestEngineWeb.beforeTest(TestEngineWeb.java:602)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:658)
	at org.testng.TestRunner.run(TestRunner.java:626)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeTest -->
  <testcase name="GS1_Login" time="0.000" classname="com.web.poc.script.GS1_Login">
    <skipped/>
  </testcase> <!-- GS1_Login -->
</testsuite> <!-- com.web.poc.script.GS1_Login -->
